### ISSUE TITLE
`Parameters` need a `unit` property

### ISSUE THREAD

**Issue by bbartley** [SD2E/opil#8]
Some Protocol Parameters will need to specify a unit.  It is not possible to use sbol.Measure for this, because Measures require both a unit and a value

**Comment by rmarkelo** [comment]
It appears that some parameters may only be given in particular units and we need to be able to verify this. Is this the idea?

**Comment by bbartley** [comment]
Yep, that's the idea.  For example, if there were a `Parameter` representing volume, we want to tell the user that the volume must be specified in microliters, as opposed to, say liters.  I don't think we need a new opil term for units though. I believe the right predicate to use here is simply `om:hasUnit`

**Comment by rmarkelo** [comment]
We could do that, although the Parameter class is not included in the domain specified for hasUnit in the Units of Measure ontology. That can cause problems only if we do OWL reasoning, which we have no plans to do right now.

**Comment by jakebeal** [comment]
Related to #135

**Comment by rmarkelo** [comment]
Hi Jake,

Just so I can keep abreast of what you're up to, what is PAML? A Google search turns up Phylogenetic Analysis by Maximum Likelihood, but I'm guessing you mean something else.

Thanks,

Rich


_________________________________

Richard Markeloff, Ph.D.
Senior Scientist
Raytheon BBN Technologies
Office: 703-284-1221
Mobile: 571-485-0189
Email: richard.markeloff@raytheon.com


________________________________

**Comment by jakebeal** [comment]
https://github.com/SD2E/paml
